provider/packet/environ.go

```diff
@@ -42,6 +46,30 @@ type environ struct {
 	packetClient *packngo.Client
 }
 
+const (
+	packetUserDataOverrides = `#!/bin/bash
+rm /etc/ssh/ssh_host_*dsa* 
+rm /etc/ssh/ssh_host_ed*
+rm /sbin/initctl
+sudo apt update
+sudo apt install -y dmidecode snapd
+set -e
+(grep ubuntu /etc/group) || groupadd ubuntu
+(id ubuntu &> /dev/null) || useradd -m ubuntu -s /bin/bash -g ubuntu
+umask 0077
+temp=$(mktemp)
+echo 'ubuntu ALL=(ALL) NOPASSWD:ALL' > $temp
+install -m 0440 $temp /etc/sudoers.d/90-juju-ubuntu
+rm $temp
+su ubuntu -c 'install -D -m 0600 /dev/null ~/.ssh/authorized_keys'
+export authorized_keys="%s"
+if [ ! -z "$authorized_keys" ]; then
+su ubuntu -c 'printf "%%s
```

Comment:
@achilleasa was this meant to be a `%%s` or a `%s`?

Reply:
I just formatted the existing template from the PR so it is more readable. I believe this is meant to be `%%s` because the template gets passed as an argument to `fmt.Sprintf` so you want to end up with `%s` for the printf statement in the output.

Reply:
I think this can be removed since we are already injecting the values on line 65